### DISCUSSION TITLE
Anyway to allow any one to access my containers but not break anything

### DISCUSSION THREAD

**Discussion by SignatureDom** [GriefPrevention/GriefPrevention#1347] [Q&A] [answered]
is there any way to allow people to access containers but not anything else
as in EVERYONE in my server not just people i trust

**Comment by bigpresh** [comment, marked as answer]
Yes, `/containertrust public`

**Reply by SignatureDom** [reply]
and would there be a way, to let them kill animals

**Reply by bigpresh** [reply]
Pretty sure that /containertrust also allows killing animals (it certainly lets them "use" them), but haven't time to double-check at present.  Give it a try - if it doesn't, then that would come under the full `/trust` (which includes breaking blocks too).

**Reply by SignatureDom** [reply]
ok thx for the help, just 1 final question... Is there a permission to let a group build in admin zones

**Reply by bigpresh** [reply]
OTTOMH, I believe admin claims can still have trust given to other players, but not groups, so you'd have to `/trust` each player on that claim individually.

I believe there's a mostly-undocumented way to give trust to anyone with a given permission (e.g. `/trust [foo.admin]`, then anyone who has the `foo.admin` permission in your (supported) permissions management plugin will be able to build there) but I don't have time right now to delve in to the code to verify how it works or try it out.  If it does work properly, though, we ought to document that, as it could be useful, especially if you have known teams of community builders etc.

You could give them the `griefprevention.adminclaims` permission, but then that would allow them to create, delete, modify and build in *all* admin claims which is probably not what you're looking for.

**Reply by RoboMWM** [reply]
I believe giving a permission a trust is supported, and idk if brackets are required (which is inconsistent lol). Unlike assigning bonus blocks, using permissions to assign trust makes more sense as it's only needed if the player is online anyways.

**Reply by Jikoo** [reply]
The brackets are required for permissions, that's consistent everywhere across GP afaik - anything else is parsed as a player (barring the keywords "all" and "public" of course).

**Reply by RoboMWM** [reply]
ah ok. I thought there was one case where it wasn't required somewhere.
